src/main/java/io/github/darealturtywurty/turtychemistry/core/network/rubbertreenetworking/ClientBoundUpdateRubberTreeTapPacket.java

```diff
@@ -0,0 +1,43 @@
+package io.github.darealturtywurty.turtychemistry.core.network.rubbertreenetworking;
+
+import io.github.darealturtywurty.turtychemistry.common.block.treeblocks.rubbertree.RubberTreeTap;
+import io.github.darealturtywurty.turtychemistry.core.init.BlockInit;
+import net.minecraft.client.Minecraft;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.fml.DistExecutor;
+import net.minecraftforge.network.NetworkEvent;
+
+import java.util.function.Supplier;
+
+public final class ClientBoundUpdateRubberTreeTapPacket {
+    public final BlockPos treeTapBlockPos;
+    public final boolean isTapping;
+
+    public ClientBoundUpdateRubberTreeTapPacket(final BlockPos pos, final boolean isTapping) {
+        this.treeTapBlockPos = pos;
+        this.isTapping = isTapping;
+    }
+
+    public ClientBoundUpdateRubberTreeTapPacket(final FriendlyByteBuf byteBuf) {
+        this(byteBuf.readBlockPos(), byteBuf.readBoolean());
+    }
+
+    public void encapsulate(final FriendlyByteBuf byteBuf) {
+        byteBuf.writeBlockPos(this.treeTapBlockPos);
+    }
+
+    public void process(Supplier<NetworkEvent.Context> networkContext) {
```

Comment:
Process is a little bit of a weird name. 
```suggestion
    public void handle(Supplier<NetworkEvent.Context> networkContext) {
```